phasmophobia_evidence_v3/widget.js

```diff
@@ -166,12 +189,21 @@ window.addEventListener('onWidgetLoad', function (obj) {
       "type": "Yurei",
       "conclusion": createGhostConclusionString(fieldData['yureiString'], 'Yurei'),
       "evidence": '010110'
-    }
+    }/*, {
```

Comment:
Maybe make enabling these optional via a "Toggle beta features" radio select. This would let people who play on Beta enable the new ghosts once they enter beta, but leave them out otherwise.

Reply:
This has been added as a "yes/no" dropdown under optionals and code edited accordingly.